airsonic-main/src/main/resources/liquibase/11.0/cue-support.xml

```diff
@@ -0,0 +1,59 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="startposition_001" author="yetangitu">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="media_file" columnName="start_position" />
+            </not>
+        </preConditions>
+        <addColumn tableName="media_file">
+            <column name="start_position" afterColumn="type" type="double" defaultValueNumeric="-1.0">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="dropuniqueindexfolderpath_001" author="yetangitu">
+        <dropIndex tableName="media_file" indexName="idx_media_file_folder_path" />
+        <rollback>
+            <createIndex tableName="media_file" unique="true" indexName="idx_media_file_folder_path">
+                <column name="folder_id"></column>
+                <column name="path"></column>
+            </createIndex>
+        </rollback>
+    </changeSet>
+    <changeSet id="adduniqueindexfolderpathstartposition_001" author="yetangitu">
+        <createIndex tableName="media_file" indexName="idx_media_file_folder_path_start_position" unique="true">
+            <column name="folder_id"/>
+            <column name="path"/>
+            <column name="start_position"/>
```

Comment:
can we call this index_start_position?

Reply:
Fine by me, less descriptive but shorter.

Reply:
I'd suggest idx_start_position instead, seeing how as all the other indexes use that format:

```
Indexes:
    "pk_media_file" PRIMARY KEY, btree (id)
    "idx_media_file_album" btree (album)
    "idx_media_file_album_artist" btree (album_artist)
    "idx_media_file_artist" btree (artist)
    "idx_media_file_created" btree (created)
    "idx_media_file_folder_path_start_position" UNIQUE, btree (folder_id, path, start_position)
    "idx_media_file_genre" btree (genre)
    "idx_media_file_last_played" btree (last_played)
    "idx_media_file_parent_path" btree (parent_path)
    "idx_media_file_play_count" btree (play_count)
    "idx_media_file_present" btree (present)
    "idx_media_file_type" btree (type)
```

Reply:
sorry we meant the column itself, not the index

instead of start_position, it'd be useful from the schema to see that it's an index_start_position

Reply:
the comment was in an inaccurate place and causing confusion. let's rectify